src/WebJobs.Script/DependencyInjection/ScriptStartupTypeLocator.cs

```diff
@@ -103,18 +103,20 @@ public async Task<IEnumerable<Type>> GetExtensionsStartupTypesAsync()
                 }
             }
 
-            bool isDotnetApp = isPrecompiledFunctionApp || IsDotnetIsolatedApp(workerConfigs);
+            bool isDotnetIsolatedApp = IsDotnetIsolatedApp(workerConfigs);
+            bool isDotnetApp = isPrecompiledFunctionApp || isDotnetIsolatedApp;
+            var isLogicApp = SystemEnvironment.Instance.IsLogicApp();
 
             if (SystemEnvironment.Instance.IsPlaceholderModeEnabled())
             {
                 // Do not move this.
                 // Calling this log statement in the placeholder mode to avoid jitting during specializtion
-                _logger.ScriptStartNotLoadingExtensionBundle("WARMUP_LOG_ONLY", bundleConfigured, isPrecompiledFunctionApp, isLegacyExtensionBundle);
+                _logger.ScriptStartNotLoadingExtensionBundle("WARMUP_LOG_ONLY", bundleConfigured, isPrecompiledFunctionApp, isLegacyExtensionBundle, isDotnetIsolatedApp, isLogicApp);
             }
 
             string baseProbingPath = null;
 
-            if (bundleConfigured && (!isDotnetApp || isLegacyExtensionBundle))
+            if (bundleConfigured && (!isDotnetApp || isLegacyExtensionBundle || isLogicApp))
```

Comment:
It would be good to log when we don't load the bundles. This would have helped provide a clear signal.

Reply:
We do log when are not loading bundle. The previous PR missed updating the conditions.  I can make it clearer in the log statement though.